File: java/IEC61970/InfIEC61970/InfOperationalLimits/EquipmentLimitSeriesComponent.java
package IEC61970.InfIEC61970.InfOperationalLimits;

import IEC61970.Base.Core.Equipment;
import IEC61970.Base.Core.IdentifiedObject;

/**
 * This represents one instance of an equipment that contributes to the
 * calculation of an operational limit.
 * @author akamath
 * @version 1.0
 * @updated 02-Jan-2024 11:40:20 PM
 */
public class EquipmentLimitSeriesComponent extends IdentifiedObject {

	/**
	 * Equipment contributing toward the series limit.   The reference here is to
	 * Equipment rather than a specific limit on the equipment so the grouiping can be
	 * reused for multiple limits of different types on the same instance of equipment.
	 */
	public Equipment Equipment;

	public EquipmentLimitSeriesComponent(){

	}
}//end EquipmentLimitSeriesComponent

File: py/IEC61970/InfIEC61970/InfOperationalLimits/EquipmentLimitSeriesComponent.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106 on Wed Jan  3 22:25:18 2024
from IEC61970.Base.Core.Equipment import Equipment
from IEC61970.Base.Core.IdentifiedObject import IdentifiedObject


class EquipmentLimitSeriesComponent(IdentifiedObject):
    """
    This represents one instance of equipment that contributes to the calculation of an operational limit.
    """
    def __init__(self):
        super().__init__()
        self.equipment = Equipment()  # Equipment contributing toward the series limit.

# end EquipmentLimitSeriesComponent
